README.rst

```diff
@@ -1087,6 +1087,11 @@ Choose from the list of available rules:
   Removes ``@param`` and ``@return`` tags that don't provide any useful
   information.
 
+  Configuration options:
+
+  - ``allow_mixed_superfluous_phpdoc`` (``bool``): whether to consider a mixed type
```

Comment:
The description is the wrong way around: the option says "allow" with default `false` (thus removing) but the description suggests that on default it is NOT considered superfluous (thus keeping).

Reply:
I've tried to make it better - if you come up with even better description, I'd like to update it :)

Reply:
I would rename the option to `allow_mixed` (just like the property in the fixer), with description:
```
whether type `mixed` without description is allowed (`true`) or considered superfluous (`false`)
```